bots/discord/dynamica/egg-dynamica.json

```diff
@@ -0,0 +1,59 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-12-12T16:55:59-05:00",
+    "name": "Dynamica",
+    "author": "sebastian.pietschner@gmail.com",
+    "description": "The dynamica discord bot",
+    "features": null,
+    "images": [
+        "ghcr.io/sebasptsch/dynamica"
```

Comment:
Please use our existing images and open a PR should something be missing as noted in 3. rule of the CONTRIBUTION.md that you've modified. 

We'd like to have control over the images instead of someone able to change it entirely at any point without notice.

Reply:
Incorporated the standard nodejs image.